cmd/application.go

```diff
@@ -132,6 +132,12 @@ var applicationDeleteCmd = &cobra.Command{
 		// Print App Information which will be deleted
 		err := printDeleteAppInfo(client, appName, currentProject)
 		checkError(err, "")
+		exists, err := application.Exists(client, appName)
+		checkError(err, "")
+		if !exists {
+			fmt.Printf("Application %v does not exist\n", appName)
```

Comment:
first rebase with master and then this,
`fmt.Printf("Application %v in project %v does not exist\n", appName, projectName)`